output_template/python/run.py

```diff
@@ -89,9 +87,33 @@ def main_test(input_image, library, config_file, max_percent_incorrect_values=0.
     # add the batch dimension
     data = np.expand_dims(data, axis=0)
 
+    if file_extension == '.so':  # Shared library
+        # load and initialize the generated shared model
+        nn = NNLib()
+        nn.load(model)
+        nn.init()
+
+    elif file_extension == '.pb':  # Protocol Buffer file
+        # only load tensorflow if user wants to use GPU
+        from lmnet.tensorflow_graph_runner import TensorflowGraphRunner
+        nn = TensorflowGraphRunner(model)
+
```

Comment:
Raise error.
```
$ python run.py -m ../../minimal_graph_with_shape.pb  -c ../models/meta.yaml -i ~/lmnet/tests/fixtures/cat.jpg
.pb
/usr/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: compiletime version 3.5 of module 'tensorflow.python.framework.fast_tensor_util' does not match runtime version 3.6
  return f(*args, **kwds)
Traceback (most recent call last):
  File "/home/wakisaka/blueoil/output_template/python/run.py", line 175, in <module>
    main()
  File "/home/wakisaka/blueoil/saved/cls_20190226171345/export/save.ckpt-27/128x128/output/python/.venv/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/wakisaka/blueoil/saved/cls_20190226171345/export/save.ckpt-27/128x128/output/python/.venv/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/wakisaka/blueoil/saved/cls_20190226171345/export/save.ckpt-27/128x128/output/python/.venv/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/wakisaka/blueoil/saved/cls_20190226171345/export/save.ckpt-27/128x128/output/python/.venv/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/wakisaka/blueoil/output_template/python/run.py", line 171, in main
    run_prediction(input_image, model, config_file)
  File "/home/wakisaka/blueoil/output_template/python/run.py", line 116, in run_prediction
    output, raw_image = _run(model, input_image, config)
  File "/home/wakisaka/blueoil/output_template/python/run.py", line 103, in _run
    output = nn.run(data)
  File "/home/wakisaka/blueoil/output_template/python/lmnet/tensorflow_graph_runner.py", line 62, in run
    return self.sess.run(self.output_op, feed_dict={self.images_placeholder: data})
AttributeError: 'NoneType' object has no attribute 'run'
```

Reply:
I forgot to put nn.init() I'll fix this.